lib/compress/zstd_cwksp.h

```diff
@@ -256,9 +261,39 @@ MEM_STATIC void* ZSTD_cwksp_reserve_object(ZSTD_cwksp* ws, size_t bytes) {
     }
     ws->objectEnd = end;
     ws->tableEnd = end;
+    ws->tableValidEnd = end;
     return start;
 }
 
+MEM_STATIC void ZSTD_cwksp_mark_tables_dirty(ZSTD_cwksp* ws) {
```

Comment:
Can we have a flag like `ZSTD_c_randomizeDirtyTables` and when it is set to true we will randomize the dirty tables?

Then we can sometimes enable it in `zstreamtest`, `fuzzer`, and in the oss-fuzz `FUZZ_setRandomParameters()`.

Reply:
As discussed offline, I'm going to hook this mechanism into msan.